CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Changelog
 
 ## [Unreleased]
+- Fixed `devtool build` to build jailer binary for `musl` only targets. Building
```

Comment:
This line should fall under a category under the Unreleased section. As @gc-plp mentioned, I think it should go into Unreleased -> Changed.

Reply:
My bad, corrected it now.